Library/Homebrew/extend/ENV/super.rb

```diff
@@ -87,6 +94,8 @@ def setup_build_environment(formula: nil, cc: nil, build_bottle: false, bottle_a
     self["HOMEBREW_DEPENDENCIES"] = determine_dependencies
     self["HOMEBREW_FORMULA_PREFIX"] = @formula.prefix unless @formula.nil?
 
+    set_debug_symbols if debug_symbols
```

Comment:
You're already setting `HOMEBREW_CCCFG` to an empty string above, maybe it's better to add `D` there to start with?

Reply:
That sounds good. I wasn't sure if other magic was happening that it needed to be `""`.

Reply:
It turns out that `determine_ccfg` is overridden here: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/extend/os/mac/extend/ENV/super.rb#L80-L85 so that wouldn't work (and no call to super either!)